javalib/src/main/scala/java/util/AbstractCollection.scala

```diff
@@ -0,0 +1,86 @@
+package java.util
+
+import scala.annotation.tailrec
+import scala.collection.immutable.{List => SList}
+
+abstract class AbstractCollection[E]() extends java.util.Collection[E] {
+	def iterator(): Iterator[E]
+	def size(): Int
```

Comment:
Please adhere to this repo's conventions. We use 2 spaces for indentation (4 inside `()`), never tabs.